gold-cc-cvc-input.html

```diff
@@ -160,6 +160,11 @@
           value: {
             type: String,
             observer: '_onValueChanged'
+          },
+
+          invalid: {
```

Comment:
This _should_ come from `PaperInputBehavior`, where it's already `notify: true:` :https://github.com/PolymerElements/paper-input/blob/master/paper-input-behavior.html#L81